homeassistant/components/rituals_perfume_genie/sensor.py

```diff
@@ -123,7 +112,6 @@ def extra_state_attributes(self):
         """Return the battery state attributes."""
         return {
             ATTR_BATTERY_LEVEL: self.coordinator.data[HUB][SENSORS][BATTERY][TITLE],
```

Comment:
Will the move of this attribute to a sensor be a separate PR?

Reply:
Yes, I will do the other comments in another PR.
I didn't want to add a new feature and code improvements in one PR.

Reply:
Perfect!